transformers/test_data/pot_join.go

```diff
@@ -0,0 +1,64 @@
+// VulcanizeDB
+// Copyright © 2019 Vulcanize
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package test_data
+
+import (
+	"math/rand"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+)
+
+var rawPotJoinLog = types.Log{
+	Address: common.HexToAddress(PotAddress()),
+	Topics: []common.Hash{
+		common.HexToHash("0x049878f300000000000000000000000000000000000000000000000000000000"),
+		common.HexToHash("0x000000000000000000000000e7bc397dbd069fc7d0109c0636d06888bb50668c"),
+		common.HexToHash("0x000000000000000000000000000000000000000000000000417fa3222791bd1a"),
+		common.HexToHash("0x0000000000000000000000000000000000000000000000000000000000000000"),
```

Comment:
These topics are from a real kovan event (the one used in integration tests), and I just thought it was weird how it has 3 topics even though we're only expecting 2. I'm probably missing some subtlety of solidity, but it seemed worth mentioning

Reply:
I _think_ this is because `LogNote` always includes 4 topics, regardless of how many topics are actually used by an individual `note` modifier. Hopefully we could (theoretically) omit this field and everything would still behave correctly.

Reply:
Yeah everything  does seem to work correctly if we omit the 3rd and 4th topics, but the part that seemed weird to me was that the 3rd topic isn't empty. In other events shaped the same way (e.g. vat heal events), the 3rd and 4th topics are all zeroes, like the 4th one is here.

Reply:
Oh wait I think `topics[2]` is the `wad` value we want. Pretty sure `topics[1]` is the `msg.sender`: https://github.com/makerdao/dss/blob/master/src/lib.sol#L37